config/ci-rocm.yaml

```
@@ -0,0 +1,296 @@
+defaults: &defaults
+  dirs:
+    code: code/{name}
+    venv: venv/{name}
+    data: data
+    runs: runs
+
+  venv:
+    type: virtualenv
+
+  voir:
+    --forward:
+      - "#stdout"
+      - "#stderr"
+      - "#end"
+      - loss
+      - compute_rate
+      - train_rate
+      - loading_rate
+      - gpudata
+      - gpu_time
+      - cpu_dispatch_time
+      - cpu_walltime
+      - tflops
+    # --loading-rate: true
+    --poll-gpu: 10
+
+
+torchvision: &torchvision
+  <<<: *defaults
+
+  group: torchvision
+  install_group: torchvision
+
+  dirs:
+    code: code/torchvision
+    venv: venv/torchvision
+
+  plan:
+    method: per_gpu
+
+  pip:
+    args:
+      - --extra-index-url
+      - https://download.pytorch.org/whl/rocm5.2/
+
+  voir:
+    --stop: 30
```

Because I had changed this in ci-cuda.

```suggestion
    --stop: 20
```